db/migrate/20150404104501_change_locale_for_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeLocaleForUsers < ActiveRecord::Migration
+  def change
+    change_column :users, :locale, :string, null: false, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.